learn/os_setup/install_gfortran.md

```diff
@@ -12,17 +12,17 @@ GFortran is the name of the [GNU Fortran project](https://gcc.gnu.org/fortran/).
 Three sources provide quick and easy way to install GFortran compiler on Windows:
 1. [http://www.equation.com](http://www.equation.com/servlet/equation.cmd?fa=fortran), provides 32 and 64-bit x86
  executables of the latest (10.2) gcc-version.
-￼2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides 32 and 64-bit x86 executables of the 9.2 gcc-version.
-￼3. [Mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 7.2 gcc-version.
+2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides 32 and 64-bit x86 executables of the 9.2 gcc-version.
+3. [Mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 7.2 gcc-version.
 
 In all the above choices, the process is straightforward—just download the installer and follow the installation wizard.
 
 ### Unix-like development on Windows
 For those familiar with a unix-like development environment, several emulation options are available on Windows each of which provide packages for gfortran:
 
 * __Cygwin:__ A runtime environment that provides POSIX compatibility to Windows;
-￼* __MSYS2:__ A collection of Unix-like development tools for compiling native Windows binaries;
-￼* __Windows Subsystem for Linux (WSL):__ An official compatibility layer for running Linux binary executables on Windows.
+* __MSYS2:__ A collection of Unix-like development tools for compiling native Windows binaries;
```

Comment:
I suggest: __MinGW-w64:__ ...

(There is more to say about all this, but that is not suitable for an introductory text like this! I have struggled with it recently, hence my emphasis on getting it least wrong)